### DISCUSSION TITLE
MQTT connection to cloud

### DISCUSSION THREAD

**Discussion by PemAnt** [jomjol/AI-on-the-edge-device#832] [Q&A]
Hi,
I'm trying to connect to MQTT in the cloud (Hivemq cloud) but so far unsuccessfully. When I’m test the connection via ubuntu so the messages go well. On log i have just FlowControll.doFlow - ClassFlowMQTT. It’s posible to conect to brocker on cloud? I have no idea how to make it work.

Thank you for your answer. 

[Snímek obrazovky 2022-06-22 215559](https://user-images.githubusercontent.com/108027517/175125155-ed36eea1-3256-4842-bc4f-8079ab737cd7.png)

**Comment by jomjol** [comment]
The mqtt is unchanged since a long time and other users do not report problems. I assume it is in your configuration (config.ini).

**Comment by veerur4u** [comment]
@PemAnt  .. were you able to resolve your issue, we are planning to use hive MQ and wanted to understand how you had resolved this issue
